### PR TITLE
Add new interface(dtmi:covia:CAT_G3;1)

### PR DESCRIPTION
**Pull request by KeijiKomaki**
### Thank you for contributing to the Azure IoT Plug and Play Models repository

This project has adopted the [Microsoft Open Source Code of Conduct](https://opensource.microsoft.com/codeofconduct/). For more information see the [Code of Conduct FAQ](https://opensource.microsoft.com/codeofconduct/faq/) or contact [opencode@microsoft.com](mailto:opencode@microsoft.com) with any additional questions or comments.

The repository pipeline will ensure minimum validation of incoming DTDL models.  

- PR validation steps are described in the tools [Wiki](https://github.com/Azure/iot-plugandplay-models-tools/wiki/Validation-Pipeline#pr-validation-checks).
- The same validation can be run locally via the [dmr-client](https://github.com/Azure/iot-plugandplay-models-tools/tree/dev/clients/dotnet#device-model-repository-client) CLI.

## Requested info template for model(s) submission

When submitting models to the repository we ask that you provide as much of the following meta information around your models and related devices as possible. This info will be used to improve Plug and Play.

:star2: Please replace the markdown comment examples with your own values.

### CODE REVIEW

**Review by rido-min — CHANGES_REQUESTED** [review]
@KeijiKomaki thanks for your contribution. 

To be consistent with all covia interfaces, can you use the same DTMI as in #22 #309, and #312 ?

instead of `dtmi:covia` use `dtmi:jp:co:covia`

File: dtmi/covia/component/location-1.json [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+            "displayName": "High accuracy",
+            "description": "It is highly accurate, but consumes a lot of battery power.",
+            "enumValue": 3
+          }
+        ]
+      },
+      "writable": true
+    },
+    {
+      "@type": "Telemetry",
```
Comment (rido-min) [inline dtmi/covia/component/location-1.json:55]: Consider use the `Location` Semantic type, and `geopoint` schema

**Comment by KeijiKomaki** [comment]
We would like to use "dtmi:covia" in the future.
We also want to delete "dtmi:jp:co:covia" in the future.
Devices that currently use "dtmi:jp:co:covia" will use "dtmi:covia" in the next update, and the information in the device catalog will be updated.
Thank you for your cooperation.

File: dtmi/covia/component/location-1.json (hunk 1 above)
Comment (KeijiKomaki) [reply dtmi/covia/component/location-1.json:55]: For compatibility, we want to use the Object type this time.

**Review by rido-min — APPROVED** [review]

**Pull request merged**